### ISSUE TITLE
sslv3 alert illegal parameter

### ISSUE THREAD

**Issue by ChaosKnightz** [nabla-c0d3/sslyze#555]
**Describe the bug**
When performing a scan against certain hosts where sslv3 is enabled, sslyze returns the following error "error:14094417:SSL routines:ssl3_read_bytes:sslv3 alert illegal parameter\n"

**To Reproduce**
ran a default scan by just calling sslyze and the targeted ip and received the following result

Note the ip has been sanitized for legal concerns, the actual host was provided directly to the creator of sslyze.
```
# sslyze x.x.x.x

 CHECKING CONNECTIVITY TO SERVER(S)
 ----------------------------------

   x.x.x.x:443        => ERROR: Unexpected connection error: "('error:14094417:SSL routines:ssl3_read_bytes:sslv3 alert illegal parameter\n',)"; discarding scan.
 SCANS COMPLETED IN 0.15696 S
```

**Expected behavior**
should have returned data from the host; however, I received an error instead. I confirmed that this host was live, and that it contained sslv3 ciphers by leveraging sslscan from the same device. Ssl scan returned data, however, sslyze errored out.

**Python environment (please complete the following information):**
 - OS: kali linux
 - Python version: 3.9.6

**Comment by nabla-c0d3** [comment]
Note for myself: I have the actual host in my Twitter DMs.

**Comment by JJK96** [comment]
I have the same issue for a host I'm scanning. Possibly the host I'm scanning requires a client certificate. Which could be a cause for the problem, as reported in the following stack overflow post: https://stackoverflow.com/a/62472367.

OS: WSL (`Linux LT446W3J3 5.10.102.1-microsoft-standard-WSL2 #1 SMP Wed Mar 2 00:30:59 UTC 2022 x86_64 GNU/Linux`)
Python version: 3.10.2

**Comment by nabla-c0d3** [comment]
Fixed in [v5.0.5](https://github.com/nabla-c0d3/sslyze/releases/tag/5.0.5).
